pkg/apis/application/v1alpha1/app_types.go

```diff
@@ -85,6 +85,8 @@ type AppSpec struct {
 
 // +k8s:openapi-gen=true
 type AppSpecConfig struct {
+	// Whether to install CRDs that might be supplied with the chart
+	SkipCRDInstall bool `json:"skipCrdInstall"`
```

Comment:
```suggestion
	SkipCRDInstall bool `json:"skipCrdInstall,omitempty"`
```

Adding omitempty will make the field optional as per my other comment.

Reply:
I am not sure about making it optional.... Doesn't a zero value boolean in go default to `false`? which is exactly what we want. It would conditional logic in the code similar right? something like...
```golang
if cr.Install.SkipCRDs {
...
}
```
Rather trying to test for the existence of the bool then test for true/false

Reply:
Yes in the operator the boolean will default to false. But one thing to watch is the install struct as it could be nil.

The omitempty is about keeping the app CRs cleaner when they are viewed in kubectl etc. So the setting will only appear if the user has requested CRDs to be skipped.